packages/common/src/ui/components/buttons/SplitButton.stories.tsx

```diff
@@ -1,28 +1,85 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Box } from '@mui/material';
 import { ComponentMeta } from '@storybook/react';
-import { SplitButton } from './SplitButton';
+import { SplitButton, SplitButtonOption } from './SplitButton';
+import { useI18N } from '@common/intl';
 
-const ops = [
-  { label: 'Create a merge commit' },
-  { label: 'Squash and merge' },
-  { label: 'Rebase and merge' },
+const ops: [
+  SplitButtonOption<string>,
+  SplitButtonOption<string>,
+  SplitButtonOption<string>
+] = [
+  { label: 'Create a merge commit', value: 'createAndMerge' },
+  { label: 'Squash and merge', value: 'squashAndMerge' },
+  { label: 'Rebase and merge', value: 'rebaseAndMerge' },
 ];
 
-const Template = () => {
+const BasicComponent = () => {
+  const [selectedOption, setSelectedOption] = useState<
+    SplitButtonOption<string>
+  >(ops[0]);
+
+  return (
+    <Box>
+      <SplitButton
+        ariaLabel="Split button"
+        ariaControlLabel="open split button menu"
+        options={ops}
+        onClick={option => alert(JSON.stringify(option))}
+        selectedOption={selectedOption}
+        onSelectOption={setSelectedOption}
+      />
+    </Box>
+  );
+};
+
+const IsDisabledComponent = () => {
+  ops[1].isDisabled = true;
+  const [selectedOption, setSelectedOption] = useState<
+    SplitButtonOption<string>
+  >(ops[0]);
+
+  return (
+    <Box>
+      <SplitButton
+        ariaLabel="Split button"
+        ariaControlLabel="open split button menu"
+        options={ops}
+        onClick={option => alert(JSON.stringify(option))}
+        selectedOption={selectedOption}
+        onSelectOption={setSelectedOption}
```

Comment:
um.. this one needs the `isDisabled` prop 🙂

Reply:
One of the options was disabled 😁  Have added an option disabled story and added the is disabled prop to this one [74cd53f](https://github.com/openmsupply/openmsupply-client/pull/820/commits/74cd53f26f7a521bdbb475e8883d122da13fcf0a)

Reply:
I thought that might be it.. but the first story also had a disabled option and I wasn't sure what the difference was supposed to be! thanks - I think I get it now :)